### PR TITLE
ADD storybook

### PR DESCRIPTION
**Pull request by ndelangen**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:
Storybook is a massively popular tool for UI developers and has a rich ecosystem of addons developed by people outside the core team. A topic would be great so users can easily find community projects.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/storybook/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+aliases: storybooks
+created_by: Storybook Team
```
Comment (nayafia) [inline topics/storybook/index.md]: Specific names would be better here, or remove this field entirely, I think (seems too self-referential).

File: topics/storybook/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+display_name: Storybook
+logo: storybook.png
+released: March 31, 2016
+short_description: The UI Development Environment you'll ♥️ to use
```
Comment (nayafia) [inline topics/storybook/index.md]: We generally try to use a more neutral tone. Could you rephrase this to something more descriptive? Even the first sentence of your longer description (_"Storybook is a UI development environment for your UI components."_) would work.

File: topics/storybook/index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+topic: storybook
+url: https://storybook.js.org
+---
+Storybook is a UI development environment for your UI components. With it, you can visualize different states of your UI components and develop them interactively.
+
+Storybook runs outside of your app. So you can develop UI components in isolation without worrying about app specific dependencies and requirements.
```
Comment (nayafia) [inline topics/storybook/index.md]: Try rephrasing this/next paragraph to something more neutral, too. It should feel more like Wikipedia and less like a marketing page :wink:

**Comment by nayafia** [comment]
Hiiii @ndelangen! :blush::wave: I left some comments in my review. Also, could you address all broken tests? (I requested other edits on the `short_description`, but the logo also needs to be resized)

File: topics/storybook/index.md (hunk 1 above)
Comment (ndelangen) [reply topics/storybook/index.md]: removed it

**Comment by ndelangen** [comment]
I think I've fixed everything! 🙇

**Comment by ndelangen** [comment]
Anything else I need to do?

**Comment by MikeMcQuaid** [comment]
Apologies for the delay @ndelangen! Thanks for your first contribution to github/explore! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by ndelangen** [comment]
Awesome!!
